src/MCPClient/lib/clientScripts/examineContents.py

```diff
@@ -12,6 +12,11 @@ def main(target, output):
     try:
         os.makedirs(output)
         subprocess.call(args)
+	# remove empty BulkExtractor logs
+	for filename in os.listdir(output):
+	    filepath = os.path.join(path,filename)
```

Comment:
Looks like this should be `filepath = os.path.join(output, filename)` since `path` is undefined.